src/main/bash/sdkman-env.sh

```diff
@@ -17,27 +17,67 @@
 #
 
 function __sdk_env() {
-	readonly sdkmanrc='.sdkmanrc'
+	SDKMANRC=".sdkmanrc"
```

Comment:
Was there a problem with `readonly`? I would even be happy with it being `local`, but don't really like it defined as global though.

Reply:
I don't know why @helpermethod mad it a global.  `readonly` variables fail in bash 3 when a method is called more than once, as `readonly` in bash 3 creates a readonly global.  The way around this is to use `local -r` which works in bash 3, as well as in bash 5 & in zsh.  If that's why he changed it, using `local -r` should fix the problem.

Reply:
I will make it local and pass it explicitly to the _generate function.

Reply:
I've mad the variable local and readonly (local - r).